### PR TITLE
Made Working Stats Command

### LINKED ISSUE #25: Server Stats Command
Should show total channels (out of Discord's max limitation), user count, users by role, class count, activity, and whatever else we can think of

### PR DESCRIPTION
**Pull request by L30N1DAS**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
List any dependencies that are required for this change.

Added a Stats command so admins can view various statistics and analyze the condition of the server.

## Issues

Closes #25 

## Type of change

Select one or more of the following:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (describe below)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. List any edge cases you tested.
If you come up with any edge cases you didn't test while writing this PR, cancel the PR and test again.

- [x] Ran the command `-stats` in both private server and CSE Testing Server
- [x] Counted number of total channels, text channels, voice channels, users, classes, roles, and users with top ten roles in CSE Testing Server and obtained expected results

# Checklist:

- [x] All local commits have been pushed to remote
- [x] All changes on the base branch have been merged into this branch, either by rebase or merge
- [x] My code is [PEP-8](https://pep8.org/) compliant (excluding maximum line length, keep that to 100ish characters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added [Google Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for all new functions/methods
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by L30N1DAS** [comment]
Thinking of adding most messaged-in channel over last 24 hrs, user who sent most messages in last 24 hrs, and most @ed user in last 24 hrs at some point

**Comment by CodingPenguin1** [comment]
> Thinking of adding most messaged-in channel over last 24 hrs, user who sent most messages in last 24 hrs, and most @ed user in last 24 hrs at some point

I like it, but I would probably do stats by semester

File: Cogs/AdminCommands.py [hunk 1]
```diff
@@ -155,3 +156,63 @@ async def stop(self, ctx):
...
+    async def stats(self, ctx):
+        """Outputs various stats of the server
+        Send message with server stats to user
+
+        Outputs:
+            Message to chat including total number of channels, text channels, voice channels, users, classes, roles,
+            and people with top ten roles.
+        """
+
+        guild=ctx.guild
```
Comment (CodingPenguin1) [inline Cogs/AdminCommands.py:171]: Minor formatting request for consistency: put spaces on either side of `=` when using it as assignment. Not needed for parameter kwargs

File: Cogs/AdminCommands.py [hunk 2]
```diff
@@ -155,3 +156,63 @@ async def stop(self, ctx):
...
+
+        embed=discord.Embed(
+        title="Server Stats",
```
Comment (CodingPenguin1) [inline Cogs/AdminCommands.py:174]: Again, minor formatting consistency: indent this line so all parameters are the same indentation level

**Review by CodingPenguin1 — CHANGES_REQUESTED** [review]
Looks really good, just a few minor formatting things I ask to change for consistency and PEP8 compliancy

**Comment by L30N1DAS** [comment]
> > Thinking of adding most messaged-in channel over last 24 hrs, user who sent most messages in last 24 hrs, and most @ed user in last 24 hrs at some point
> 
> I like it, but I would probably do stats by semester

hmmm, good idea, I'll look into how to do that

**Review by mbozada — APPROVED** [review]
Looks good. Anmol demoed the command to us.
